src/textual/screen.py

```diff
@@ -94,11 +94,12 @@ def __call__(self, result: ScreenResultType) -> None:
 class Screen(Generic[ScreenResultType], Widget):
     """The base class for screens."""
 
-    AUTO_FOCUS: ClassVar[str | None] = "*"
+    AUTO_FOCUS: ClassVar[str | None] = None
```

Comment:
Feel like I'm very late to the party with this, but out of curiosity - since this is a ClassVar do we not have a way of having 2 Screen instances have different auto-focus depending on context?

For example, when you push an EditForm screen, maybe the initially focused form control differs based on how you arrived at that screen. If you clicked "edit username", the username input would be auto-focused when the screen is pushed. If you clicked "edit date of birth", the date of birth input would be auto-focused when the screen is pushed.

Reply:
In that case you could set `AUTO_FOCUS` to `""` (to disable it), then handle focusing manually.

Reply:
You can also set `AUTO_FOCUS` on the instance and it'll be picked up. 🤷 
Or what am I missing?